client/client.go

```diff
@@ -1131,6 +1143,74 @@ func (c *Client) run() {
 	}
 }
 
+// submitNodeEvents is used to submit a client-side node event. Examples of
+// these kinds of events include when a driver moves from healthy to unhealthy
+// (and vice versa)
+func (c *Client) submitNodeEvents(events []*structs.NodeEvent) error {
+	nodeID := c.Node().ID
+	nodeEvents := map[string][]*structs.NodeEvent{
+		nodeID: events,
+	}
+	req := structs.EmitNodeEventsRequest{
+		NodeEvents:   nodeEvents,
+		WriteRequest: structs.WriteRequest{Region: c.Region()},
+	}
+	var resp structs.EmitNodeEventsResponse
+	if err := c.RPC("Node.EmitEvents", &req, &resp); err != nil {
+		return fmt.Errorf("Emitting node event failed: %v", err)
+	}
+	c.logger.Printf("[INFO] client: emit node events complete")
```

Comment:
Not a useful info level log. These should be useful for operators. Can you remove this